### PR TITLE
Validate fix

### PR DESCRIPTION
**Pull request by jshum2479**

### CODE REVIEW

File: core/src/main/python/validate.py [hunk 1]
```diff
@@ -245,7 +246,11 @@ def main(args):
                     sys.exit(CommandLineArgUtil.PROG_ERROR_EXIT_CODE)
                 elif validation_results.get_warnings_count() > 0:
                     cla_helper.clean_up_temp_files()
-                    sys.exit(CommandLineArgUtil.PROG_WARNING_EXIT_CODE)
+                    if model_context.get_validation_method() is 'strict':
+                        sys.exit(CommandLineArgUtil.PROG_WARNING_EXIT_CODE)
+                    else:
+                        sys.exit(CommandLineArgUtil.PROG_OK_EXIT_CODE)
+
```
[the remark this answers is not in the document]
Comment (ddsharpe) [reply core/src/main/python/validate.py]: agreed.  lax means do not check, so I wouldn't warn.  you could print an info that it was seen and skipped, but otherwise no warning is necessary.

[the remark this answers is not in the document]
Comment (tbarnes-us) [reply core/src/main/python/validate.py]: Agreed. Info is better when missing files are expected - and the adminstrator has already indicated this via the command line option. The only warnings should be those that may need investigation.

[the remark this answers is not in the document]
Comment (jshum2479) [reply core/src/main/python/validate.py]: probably ok, I will print as info level but the exit code is always 0 even if there is warning - I don't know where else or if there is other place for warning code

[the remark this answers is not in the document]
Comment (rakillen) [reply core/src/main/python/validate.py]: I think the exit code should be non-zero when there is a valid warning, but I'm not sure if there are other warnings that will cross up image tool?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,7 +1,6 @@
 pipeline {
     agent {
         docker {
-            alwaysPull true
```
Comment (ddsharpe) [inline Jenkinsfile]: Please revert this change.  It was a temporary attempt to get around the OCIR outage.

File: core/src/main/python/validate.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddsharpe) [reply core/src/main/python/validate.py]: if a warning is found, it should have the CommandLineArgUtil.PROG_WARNING_EXIT_CODE

**Comment by jshum2479** [comment]
changes reverted
